blocks/secure-assets/secure-assets.css

```diff
@@ -0,0 +1,8 @@
+.block.secure-assets > div {
+  display: flex;
+  gap: 2rem;
+}
+
+.block.secure-assets img {
+  width: 43rem;
```

Comment:
Can we omit this style? Seems very project specific.

Reply:
removed the specific style.